<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminView</class>
 <widget class="QDialog" name="AdminView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="ItemTable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>511</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>182</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Item ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_DeleteId"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="BorrowedItem">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>511</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Error">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>660</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Avaliable Items:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrowed Items:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_LoadTable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>620</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AddItem">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>620</y>
     <width>97</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>620</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Sendmail">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>620</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop SendMail</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
